### ISSUE TITLE
Better exception viewing in Kibana?

### ISSUE THREAD

**Issue by vaindil** [serilog/serilog-sinks-elasticsearch#79]
I'm guessing this isn't a problem with this sink, but I'm really not sure. Exceptions, mainly stack traces, are a disaster to view in Kibana. Right now I have to copy/paste into Sublime Text and find/replace \n with a line break. If that's how I have to do it then it's no problem, but I wanted to check if there's a better way to do it.

![capture](https://cloud.githubusercontent.com/assets/771134/21472492/4edb40a0-caab-11e6-82a1-f044a939deb0.PNG)

**Comment by diegofrata** [comment]
Would be nice to have an option flag to post the exception as text instead of object.

**Comment by mivano** [comment]
You have the option to specify your own formatter. That way you are in control what you want to output.

**Comment by svickers** [comment]
Using the custom formatter makes the display drastically better.

`CustomFormatter = new ExceptionAsObjectJsonFormatter(renderMessage:true)`

<img width="1356" alt="screen shot 2017-04-10 at 9 48 01 am" src="https://cloud.githubusercontent.com/assets/105884/24872614/d761afd4-1dd2-11e7-92b1-51d940bd4cf1.png">

**Comment by davidfowl** [comment]
I just ran into this. Is there any reason this isn't the default?

![image](https://user-images.githubusercontent.com/95136/28111035-f874973a-66a8-11e7-97c9-96f47885c1b6.png)

**Comment by Mpdreamz** [comment]
We had some discussion on this here: https://github.com/serilog/serilog-sinks-elasticsearch/pull/41

Tldr it was one breaking change too many at the time, maybe something for the next major?

**Comment by mivano** [comment]
There have been many changes to the way exceptions are outputted. Closing this old issue.
